spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/configuration/CommonClientConfiguration.java

```diff
@@ -44,6 +46,7 @@ public class CommonClientConfiguration {
   ZeebeSelfManagedProperties zeebeSelfManagedProperties;
 
   @Bean
+  @ConditionalOnMissingBean
```

Comment:
❓ Why this annotation is needed ?

Reply:
So customers can provide their own definition for bean and in such case this method will be completely ignored.